apps/backend/src/main.ts

```diff
@@ -49,30 +49,31 @@ async function bootstrap() {
   );
   app.use(json({limit: '50mb'}));
   app.use(passport.initialize());
-  // Sessions are only used for oauth callbacks
-  if (configService.enabledOauthStrategies().length) {
```

Comment:
With taking out this check, do we need to account for any other error logic here?